scss/detail.scss

```diff
@@ -5,8 +5,9 @@
 }
 
 .all-books-list {
+  background-color: $wild-sand;
   display: flex;
-  justify-content: center;
+  width: 1365px;
```

Comment:
I don't understand this value can you explain me?
If you computer has less resolution you are creating scrolling on x axis

Reply:
You are right, I was using that because it was the width that Invision specifies, but if I remove it is better! It was creating scrolling unnecessarily!